src/exe.ml

```diff
@@ -124,6 +134,12 @@ let link_exe
       ?(js_of_ocaml=Dune_file.Js_of_ocaml.default)
       cctx
   =
+  let linkage, js =
+    if linkage.ext = ".bc.js" then
+      {linkage with ext = ".bc"}, true
+    else
+      linkage, false
+  in
```

Comment:
What happens if we use `(modes byte s)`? It seems to me that the rule for `.bc` is going to be produced twice